config/routes.rb

```diff
@@ -5,7 +5,8 @@
     resources :users, :destinations, :activities, :trips, :extras
   end
   get '/admin/dashboard', to: 'admin/dashboard#index'
-  resources :users, only: [:new, :create]
+  resources :users, only: [:new, :create, :show]
+  get '/profile', to: 'users#show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.